dirty_cat/datasets/_tests/test_fetching.py

```diff
@@ -25,9 +25,11 @@
     _get_features,
     _read_json_from_gz,
 )
-from dirty_cat.datasets.fetching import fetch_openml_dataset as _fetch_openml_dataset
-from dirty_cat.datasets.fetching import fetch_world_bank_indicator
-from dirty_cat.datasets.utils import get_data_dir as _get_data_dir
+from dirty_cat.datasets._fetching import fetch_openml_dataset as _fetch_openml_dataset
+from dirty_cat.datasets._fetching import (
+    fetch_world_bank_indicator as _fetch_world_bank_indicator,
+)
+from dirty_cat.datasets._utils import get_data_dir as _get_data_dir
```

Comment:
The imports do not need to be private, as the `_tests` directory is private.

Reply:
Ok, it was like this before. Will change.